src/roiextractors/segmentationextractor.py

```diff
@@ -231,18 +230,21 @@ def get_sampling_frequency(self):
 
         Returns
         -------
-        samp_freq: float
-            Sampling frequency of the recordings in Hz.
+        sampling_frequency: float
+            Sampling frequency of the recording in Hz.
         """
-        return float(self._sampling_frequency)
+        if self._sampling_frequency is not None:
+            return float(self._sampling_frequency)
+
+        return self._sampling_frequency
```

Comment:
Should I rollback this change @CodyCBakerPhD ?

Reply:
If I recall, the approach we took with ImagingExtractors was to define it at each format level: https://github.com/catalystneuro/roiextractors/blob/master/src/roiextractors/imagingextractor.py#L36 and assert in every format specific test that it is, indeed, a float https://github.com/catalystneuro/roiextractors/blob/52f4704c0148ce2a95fd6294ed68677b235fc056/src/roiextractors/testing.py#L353

So yeah I'd say let's synchronize this with the segmentation. It can be in a separate PR though, this one can just establish the frame_to_time